force-app/main/default/lwc/wireGetRecord/wireGetRecord.js

```diff
@@ -0,0 +1,18 @@
+import { LightningElement, wire } from 'lwc';
+import { getRecord } from 'lightning/uiRecordApi';
+import NAME_FIELD from '@salesforce/schema/User.Name';
+import EMAIL_FIELD from '@salesforce/schema/User.Email';
+
+export default class WireGetRecord extends LightningElement {
+    @wire(getRecord, { 
+        recordId: '005xx000001X8BdAAK', 
```

Comment:
Hard-coded value has to be removed. Instead, you can use @salesforce/user/Id. Something like this--
import Id from '@salesforce/user/Id';
......
......
userId = Id;
    @wire(getRecord, { 
        recordId: '$userId', 
        fields: [NAME_FIELD], 
        optionalFields: [EMAIL_FIELD]  
    }) record;
.....
.......